_episodes/15.md

```diff
@@ -0,0 +1,367 @@
+rerun tests through Github without write permissions to the Trino repository, so
+you have to do a dummy commit.
+
+This can easily be done using this one line hack 
+`git commit --amend --no-edit && git push -f`.
+
+The good news is, we have gone through some extensive lengths to identify flaky
+tests in the last year. These test failures are much rarer now, and we are 
+constantly improving the build stability as an ongoing effort.
+
+## Events, news, and various links
+
+### WTD Portland
+
+Interested in supporting the Trino project, but don't know where to start? A 
+good place to start with a little less barrier to entry, is adding to the 
+documentation. We will be supporting the 
+[writing day](https://trino.io/blog/2021/04/14/wtd-writing-day.html) at the
+Write the Docs (WTD) Portland conference this April! Join us to learn how to get involved!
+
+### Virtual Trino meetups
+
+Come join us for the inaugural Virtual Trino meetup on April 21st in the virtual
+meetup group in your region! See [the community page](./community.html) for more
+details.
+
+ - [Trino Americas meetup](https://www.meetup.com/trino-americas/events/277246268/)
+ - [Trino EMEA meetup](https://www.meetup.com/trino-emea/events/277246173/)
+ - [Trino APAC meetup](https://www.meetup.com/trino-apac/events/277246078/)
+
+At these meetups, the four Trino/Presto founders will be updating everyone on 
+the state of Trino. We'll discuss the rebrand, talk about the recent features, 
+and discuss the trajectory of the project. Then we will host a hangout and an
+ask me anything (AMA) session. Hope to see you all there!
+
+Blogs
+ - <https://medium.com/expedia-group-tech/a-short-introduction-to-apache-iceberg-d34f628b6799>
+ - <https://engineering.linkedin.com/blog/2021/fastingest-low-latency-gobblin>
+ - <https://medium.com/adobetech/iceberg-at-adobe-88cf1950e866>
+ - <https://medium.com/adobetech/high-throughput-ingestion-with-iceberg-ccf7877a413f>
+ - <https://medium.com/adobetech/taking-query-optimizations-to-the-next-level-with-iceberg-6c968b83cd6f>
+ - <https://thenewstack.io/apache-iceberg-a-different-table-design-for-big-data/>
+
+Videos
+ - <https://www.youtube.com/watch?v=mf8Hb0coI6o>
+ - <https://www.youtube.com/watch?v=Kxbzr7UP1dI&t=1274s>
+ - <https://www.youtube.com/watch?v=QNmSXMQ-gY4>
+
+Trino Meetup groups
+ - Virtual
+   - [Virtual Trino Americas](https://www.meetup.com/trino-americas/)
+   - [Virtual Trino EMEA](https://www.meetup.com/trino-emea/)
+   - [Virtual Trino APAC](https://www.meetup.com/trino-apac/)
+ - East Coast (US)
+   - <https://www.meetup.com/trino-boston/>
+   - <https://www.meetup.com/trino-nyc/>
+ - West Coast (US)
+   - <https://www.meetup.com/trino-san-francisco/>
+   - <https://www.meetup.com/trino-los-angeles/>
+ - Mid West (US)
+   - <https://www.meetup.com/trino-chicago/>
+
+Latest training from David, Dain, and Martin(Now with timestamps!):
+ - <https://trino.io/blog/2020/07/15/training-advanced-sql.html>
```

Comment:
probably better to use post_link or at least site.url liquid tag and also give the link an actual title each